src/main/java/com/google/step/finscholar/data/College.java

```diff
@@ -0,0 +1,163 @@
+public class College {
+
+  private String schoolName;
+  private UUID collegeUuid;
+  private String institutionType;
+  private double acceptanceRate;
+  private double averageACTScore;
+  private List<UUID> usersUUIDList;
+  private int totalCostAttendance;
+  private int netCostForFirstQuintile; // $0-$30,000.
+  private int netCostForSecondQuintile; // $30,001-$48,000.
+  private int netCostForThirdQuintile;  // $48,001-$75,000.
+  private int netCostForFourthQuintile; // $75,001-$110,000.
+  private int netCostForFifthQuintile; //$110,000+.
+  private int cumulativeMedianDebt;
+
+  // Constructor.
+
+  public College(String schoolName) {
+    this.schoolName = schoolName;
+    this.collegeUuid = generateUUID();
+  }
+
+  // Private helper methods.
+
+  private UUID generateUUID() {
+    return UUID.randomUUID();
+  }
+
+  // Getter methods.
+
+  /**
+   * @return - Name of the college.
+   */
+  public String getSchoolName() { return this.schoolName; }
+
+  /**
+   * @return - UUID for the college.
+   */
+  public UUID getCollegeUuid() { return this.collegeUuid; }
+
+  /**
+   * @return - Institutiton Type: Public, private, for-profit, etc.
+   */
+  public String getInstitutionType() { return this.institutionType; }
+
+  /**
+   * @return - Acceptance Rate - Total Accepted/Total applicants.
+   */
+  public double getAcceptanceRate() { return this.acceptanceRate; }
+
+  /**
+   * @return - Median ACT Score of students accepted.
+   */
+  public double getAverageACTScore() { return this.averageACTScore; }
+
+  /**
+   * @return - Users associated with the college.
+   */
+  public List<UUID> getUsersUUIDList() { return this.usersUUIDList; }
+
+  /**
+   * @return - Average total cost of attendance without financial Aid.
+   */
+  public int getTotalCostAttendance() { return this.totalCostAttendance; }
+
+  /**
+   * @return - Average net cost for the income quintile between $0-$30,000.
+   */
+  public int getNetCostForFirstQuintile() { return this.netCostForFirstQuintile; }
+
+  /**
+   * @return - Average net cost for the income quintile between $30,001-$48,000.
+   */
+  public int getNetCostForSecondQuintile() { return this.netCostForSecondQuintile; }
+
+  /**
+   * @return - Average net cost for the income quintile between $48,001-$75,000.
+   */
+  public int getNetCostForThirdQuintile() { return this.netCostForThirdQuintile; }
+
+  /**
+   * @return - Average net cost for the income quintile between $75,001-$110,000.
+   */
+  public int getNetCostForFourthQuintile() { return this.netCostForFourthQuintile; }
+
+  /**
+   * @return - Average net cost for the income quintile between $110,000+.
+   */
+  public int getNetCostForFifthQuintile() { return this.netCostForFifthQuintile; }
+
+  // Setter methods.
+
+  /**
+   * @param newInstitutionType - Public, private, for-profit, etc.
+   */
+  public void setInstitutionType(String newInstitutionType) { this.institutionType = newInstitutionType.toLowerCase(); }
```

Comment:
Consider making your setters return the "this" object instead of void. This would allow you to chain setters during initialization